package/doc/sphinx/source/documentation_pages/analysis/hole2.rst

```diff
@@ -1,35 +1,282 @@
-===================================================
- HOLE analysis --- :mod:`MDAnalysis.analysis.hole2`
-===================================================
+"""HOLE Analysis --- :mod:`MDAnalysis.analysis.hole2`
+=====================================================================================
 
 :Author: Lily Wang
 :Year: 2020
 :Copyright: GNU Public License v3
 
 .. versionadded:: 1.0.0
 
-This module provides an updated interface for the HOLE_ suite of tools [Smart1993]_
-[Smart1996]_ to analyse an ion channel pore or transporter pathway [Stelzl2014]_
-as a function of time or arbitrary order parameters. It replaces :mod:`MDAnalysis.analysis.hole`.
+This module contains the tools to interface with HOLE_ [Smart1993]_
+[Smart1996]_ to analyse an ion channel pore or transporter pathway [Stelzl2014]_ .
 
-HOLE_ must be installed separately and can be obtained in binary form
-from http://www.holeprogram.org/ or as source from
-https://github.com/osmart/hole2. (HOLE is open source and available
-under the Apache v2.0 license.)
+Using HOLE on a PDB file
+------------------------
 
-Module
-------
+Use the :func:``hole`` function to run `HOLE`_ on a single PDB file. For example,
+the code below runs the `HOLE`_ program installed at '~/hole2/exe/hole' ::
 
-.. automodule:: MDAnalysis.analysis.hole2.hole
-		
+    from MDAnalysis.tests.datafiles import PDB_HOLE
```

Comment:
It looks to me like you've just replaced the contents of this file with the docs from `hole2.hole`. Could you explain how this fixes issue #2786 and improves the documentation?

If you look at my earlier review, you'll see I recommended replacing `.. automodule:: MDAnalysis.analysis.hole2.hole` with `.. automodule:: MDAnalysis.analysis.hole2`. Maybe this is a better solution, but could you please explain why you think it is?

For example, the deleted information about how HOLE must be installed separately and where it can be downloaded is quite important. Should it be removed?